sevntu-checks/src/test/resources/com/github/sevntu/checkstyle/checks/coding/FieldDeclarationsCountCheckNOK.java

```diff
@@ -0,0 +1,12 @@
+package com.github.sevntu.checkstyle.checks.coding;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class FieldDeclarationsCountCheckNOK {
```

Comment:
sorry, I meant inner classes. I did a comment one more time in my second iteration of review.